ember-cli-build.js

```diff
@@ -126,25 +124,7 @@ module.exports = function(defaults) {
     // along with the exports of each module as its value.
 
     // osf-style
-    app.import(path.join(app.bowerDirectory, 'loaders.css/loaders.min.css'));
-
-    // app.import('bower_components/dropzone/dist/dropzone.js');
-    app.import({
-        development: path.join(app.bowerDirectory, 'dropzone/dist/dropzone.css'),
```

Comment:
Do these imports need to happen? Not from bower, obviously, but will making them unavailable break anything?

Reply:
Almost everything in registries isn't used. Dropzone is only needed on the preprints submission form which doesn't exist in registries. That being said it will need a full regression by QA to make sure we didn't remove anything that's actually used.